doc/source/developing/testing.rst

```diff
@@ -3,503 +3,400 @@
 Testing
 =======
 
-yt includes a testing suite which one can run on the codebase to assure that no
-breaks in functionality have occurred.  This testing suite is based on the Nose_
-testing framework.  The suite consists of two components, unit tests and answer
-tests. Unit tests confirm that an isolated piece of functionality runs without
-failure for inputs with known correct outputs.  Answer tests verify the
-integration and compatibility of the individual code unit by generating output
-from user-visible yt functions and comparing and matching the results against
-outputs of the same function produced using older versions of the yt codebase.
-This ensures consistency in results as development proceeds.
-
-.. _nosetests:
+yt includes a testing suite that one can run on the code base to assure that no
+breaks in functionality have occurred. This suite is based on the `pytest <https://docs.pytest.org/en/stable/>`_ testing framework and consists of two types of tests:
+
+* Unit tests. These confirm that an isolated piece of functionality runs without failure for inputs with known correct outputs.
+
+* Answer tests. These generate output from user-visible yt functions and compare this output against the output produced using an older, "known good", version of yt.
+
+These tests ensure consistency in results as development proceeds.
 
 The testing suite should be run locally by developers to make sure they aren't
-checking in any code that breaks existing functionality.  To further this goal,
-an automatic buildbot runs the test suite after each code commit to confirm
-that yt hasn't broken recently.  To supplement this effort, we also maintain a
-`continuous integration server <https://tests.yt-project.org>`_ that runs the
```

Comment:
why was this deleted? We still run on jenkins.

Reply:
My bad. I readded it.